applications/system/findmy/findmy_state.c

```diff
@@ -42,6 +42,13 @@ bool findmy_state_load(FindMyState* out_state) {
             }
             state.tag_type = tmp;
 
+            if(!flipper_format_read_uint32(file, "battery_level", &tmp, 1)) {
+                tmp = 0x00; // Default battery level set to Full
+                flipper_format_rewind(file);
+            }
+            FURI_LOG_I("findmy_load", "Saved Battery: %ld", tmp);
+            state.battery_level = tmp;
```

Comment:
is it really necessary to save the battery level in the config? if youre gonna update the data with the current battery level at runtime, and its stored in the data anyway, this seems redundant

Reply:
Not necessary, and I removed it for a little while. I added it back because I liked having it saved for debugging reasons but can be removed.